cmake/kokkos_tpls.cmake

```diff
@@ -46,6 +46,7 @@ ELSE()
   SET(ROCM_DEFAULT OFF)
 ENDIF()
 KOKKOS_TPL_OPTION(ROCM    ${ROCM_DEFAULT})
+KOKKOS_TPL_OPTION(ONEDPL  ${Kokkos_ENABLE_SYCL})
```

Comment:
Does that make that if enabling SYCL, configuration would fail if oneDPL cannot be found unless explicitly disabled?  (I don't have an issue with it, just making sure I understand what this does)

Reply:
Yes, that is what it's doing.